util/deploy-to-maven-central.sh

```diff
@@ -50,5 +50,5 @@ for generated_pom_file in dagger*pom.xml; do
   rm "${generated_pom_file}.asc"
 done
 
-git tag dagger-"${version_name}"
-git push --tags origin master
+git tag -a -m "Dagger ${version_name}" dagger-"${version_name}"
+git push origin tag "${version_name}"
```

Comment:
Tag name is wrong here; missing the `dagger-` prefix

Reply:
Shoot, thanks for that catch. Fixing now.